src/components/PassphraseView.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState, useEffect } from 'react'
+import PassphraseForm from 'components/PassphraseForm'
+import Passphrase2FA from 'components/2FA/Passphrase2FA'
+import classNames from 'classnames'
+import viewStyles from 'styles/view'
+
+const PasswordView = props => {
+  const {
+    onPassphraseSimpleSubmit,
+    onPassphrase2FAStep1,
+    onPassphrase2FAStep2,
+    passphrase,
+    isTwoFactorEnabled,
+    instance,
+    fetchInfos
+  } = props
+
+  useEffect(() => {
+    fetchInfos()
+  }, [])
```

Comment:
It is a bit surprising to have the component call fetchInfos but to see not update of state in this component. It seems like the parent is holding the "infos" but in this case, why is not the parent that does fetchInfos ?